DBZ-9675: Remove hard line breaks from connector documentation solved… (#7116)
debezium/debezium: documentation/modules/ROOT/pages/connectors/mariadb.adoc

For more information, see xref:{context}-purges-binlog-files-used-by-debezium[{connector-name} purges binlog files].

|`log_bin_compress`

> |Whether or not the binary log can be compressed. The {prodname} {connector-name} connector does not support compressed binary log entries, so `log_bin_compress` must be set to `0` (the default), which means no compression.

Reviewer: ```suggestion
|Specifies whether the binary log can be compressed. 
The {prodname} {connector-name} connector does not support compressed binary log entries, so for compatibility, you must set `log_bin_compress` to `0` (the default) to disable compression.
```
